packages/agents/cartographer/poller/src/lib/operations/transfers.ts

```diff
@@ -161,32 +162,44 @@ export const updateTransfers = async () => {
   for (const originDomain of domains) {
     for (const destinationDomain of domains) {
       if (originDomain == destinationDomain) continue;
-      const pendingTransfers = await database.getPendingTransfersByDomains(
-        originDomain,
-        destinationDomain,
-        100,
-        0,
-        "ASC",
-      );
-
-      const _destinationPendingQueryMetaParams: Map<string, SubgraphQueryByTransferIDsMetaParams> = new Map();
-      _destinationPendingQueryMetaParams.set(destinationDomain, {
-        maxBlockNumber: lastestBlockNumbers.get(originDomain)!,
-        transferIDs: pendingTransfers,
-      });
-      const destinationTransfers = await subgraph.getDestinationTransfersById(_destinationPendingQueryMetaParams);
 
-      if (destinationTransfers.length > 0) {
-        logger.info("Retrieved destination transfers by id", requestContext, methodContext, {
+      const destinationTransfers: XTransfer[] = [];
+      let offset = 0;
+      const limit = 100;
```

Comment:
This limit should be 1000.

Reply:
https://github.com/connext/monorepo/pull/4497/commits/faec12443969345507587917a1d0781cd0a0c635